### PR TITLE
#899: style "verify email" email for keycloak

### PR DESCRIPTION
**Pull request by samrichca**
TODO: final text

---

This PR creates a styled template with i18n for Keycloak emails.

- Add template, broken down into components
- Add English & French text
- Add "verify your email" email

Testing instructions:
- Copy the entire theme folder into Keycloak: `docker cp ./keycloak-theme KEYCLOAK_CONTAINER_ID:/opt/bitnami/keycloak/themes`
- Configure email theme in Keycloak admin.
- Register a new user in consent UI.
- Check Mailhog for your "verify your email" email.

### CODE REVIEW

File: email/messages/messages_en.properties [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# encoding: UTF-8
+# needed to make accents work.
+# see https://blog.mrhaki.com/2020/10/enforce-encoding-i18n-property-files-in.html
```
Comment (mistryrn) [inline email/messages/messages_en.properties:3]: Nice find!

File: email/theme.properties [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# for keycloak - don't edit keys
+locales=en
+
+# custom
+
+## images
+bannerVerify=ohcrn-email_banner-verify.jpg
+baseImageUrl=http://localhost:8087/img/
```
Comment (mistryrn) [inline email/theme.properties:8]: One more thing we should bring up tomorrow during planning: we should create a ticket to track the work of putting all the email-related images somewhere in the main OHCRN site's assets, and then updating these `baseImageUrl`s

**Review by mistryrn — APPROVED** [review]
Thanks for all the hard work and trial and error/investigating to get the templates to play nice with Keycloak/FreeMarker! Tested locally and it works great!

We'll need to figure out a plan for deploying this to Dev and beyond, hopefully we can get some of Ann and Justin's time to chat about that.

**Review by shailikadakia-oicr — APPROVED** [review]

File: email/theme.properties (hunk 2 above)
Comment (samrichca) [reply email/theme.properties:8]: https://github.com/OHCRN/platform/issues/1031

**Pull request merged**